client/app/bundles/buttons/button.scss

```diff
@@ -0,0 +1,79 @@
+@import "base.scss";
+
+
+.buttonFilledM{
+         font-size: 18px;
+         color: #6D0839;
+         font-weight: bold;
+         background-color: #FFFFFF;
+         padding-top: 17px;
+         padding-bottom: 16px;
+         padding-left: 85px;
+         padding-right: 85px;
+         border-radius: 5px;
+         border-color: #FFFFFF;
+         border-width: 0px;
+         cursor: pointer;
+ }
+
+.buttonGhostM{
+         font-size: 18px;
+         color: #FFFFFF;
+         font-weight : bold;
+         padding-top: 17px;
+         padding-bottom: 16px;
+         padding-left: 48px;
+         padding-right: 47px;
+         border-radius: 5px;
+         background-color: rgba(255,255,255,0.95);
```

Comment:
Ah, this was my mistake with my previous suggestion! Should be 10%, not 95%, sorry!